ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -146,18 +155,25 @@ pub extern "C" fn init_with_egl(
     opts: CInitOptions,
     wakeup: extern "C" fn(),
     callbacks: CHostCallbacks,
-) {
-    init_logger();
-    let gl = gl_glue::egl::init().unwrap();
-    unsafe {
-        init(
-            opts,
-            gl.gl_wrapper,
-            Some(gl.gl_context),
-            Some(gl.display),
-            wakeup,
-            callbacks,
-        )
+) -> bool {
+    let _result = panic::catch_unwind(|| {
+        init_logger();
+        let gl = gl_glue::egl::init().unwrap();
+        unsafe {
+            init(
+                opts,
+                gl.gl_wrapper,
+                Some(gl.gl_context),
+                Some(gl.display),
+                wakeup,
+                callbacks,
+            )
+        }
+    });
+
+    match _result {
+        Ok(_) => return true,
```

Comment:
nit: we can just use `result.is_ok()` (Same nit for other pattern matchings with return boolean below)

btw, maybe we don't need `underscore` prefix for `result`? 👀